src/ui.form-date-picker/composables/attrs.composable.js

```diff
@@ -1,80 +1,98 @@
 
-  const cvaCalendarWrapper = cva({
-    base: calendar.wrapper.base,
-    variants: calendar.wrapper.variants,
-    compoundVariants: calendar.wrapper.compoundVariants,
-  });
-
   const calendarWrapperClasses = computed(() =>
-    cvaCalendarWrapper({
+    cva(config.value.calendar.wrapper)({
       openDirectionY: openDirectionY.value,
       openDirectionX: openDirectionX.value,
     }),
   );
 
-  const calendarAttrs = getAttrs("calendar", {
-    isComponent: true,
-    classes: calendarWrapperClasses,
-  });
-  const inputBlurAttrs = getAttrs("input");
-  const inputActiveAttrs = getAttrs("inputActive");
-  const wrapperAttrs = getAttrs("wrapper");
-
-  const inputAttrs = computed(() => {
-    return isShownCalendar.value ? inputActiveAttrs.value : inputBlurAttrs.value;
-  });
-
-  // This watcher rewrites default calendar locales with datepicker range locales
-  // Watcher will not rewrite custom calendar locales
-  watchEffect(() => {
-    if (!calendarAttrs.value.config) {
-      calendarAttrs.value.config = {};
-    }
+  for (const key in defaultConfig) {
+    if (isSystemKey(key)) continue;
 
-    if (calendarAttrs.value.config.i18n || !props.config.i18n) {
-      return;
-    }
+    const classes = computed(() => {
+      const value = config.value[key];
+
+      if (value.variants || value.compoundVariants) {
+        return cva(value)({
+          ...props,
+        });
+      }
+
+      return "";
+    });
+
+    attrs[`${key}Attrs`] = getAttrs(key, { classes });
+
+    if (key === "input") {
+      const inputActiveAttrs = getAttrs("inputActive", { classes });
+      const inputBlurAttrs = getAttrs("inputBlur", { classes });
 
-    calendarAttrs.value.config.i18n = {
-      ...config.value.i18n,
-      weekdays: {
-        shorthand: { ...config.value.i18n.weekdays.shorthand },
-        longhand: { ...config.value.i18n.weekdays.longhand },
-      },
-      months: {
-        shorthand: { ...config.value.i18n.months.shorthand },
-        longhand: { ...config.value.i18n.months.longhand },
-      },
-    };
-
-    if (props.config.i18n.weekdays.userFormat) {
-      calendarAttrs.value.config.i18n.userFormat = {
-        ...config.value.i18n.weekdays.userFormat,
-      };
+      attrs[`${key}Attrs`] = computed(() => {
+        return isShownCalendar.value ? inputActiveAttrs.value : inputBlurAttrs.value;
+      });
     }
 
-    if (props.config.i18n.months.userFormat) {
-      calendarAttrs.value.config.i18n.userFormat = { ...config.value.i18n.months.userFormat };
+    if (key === "calendar") {
+      const calendarAttrs = attrs[`${key}Attrs`];
+
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...calendarAttrs.value,
+        class: cx([calendarWrapperClasses.value, calendarAttrs.value.class]),
+      }));
+
+      // This watcher rewrites default calendar locales with datepicker range locales
+      // Watcher will not rewrite custom calendar locales
+      watchEffect(() => {
+        if (!calendarAttrs.value.config) {
```

Comment:
here and in all places below you need to modify not cashed variable `calendarAttrs` but current variable **attrs[`${key}Attrs`]**